optimizely/3540700460-sem-campaign.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Experiment ID: 3540700460
+ * Description: SEM campaign effectiveness
+ * Mana URL: https://mana.mozilla.org/wiki/display/FIREFOX/%5Bbug+1205745%5D+SEM+campaign+effectiveness
+ */
+
+// Regexp used for targeting
+function snippetRegexp() {
+    'use strict';
+    return /^.*www\.mozilla\.org\/.+\/firefox\/new\/.+utm_source=google.+utm_medium=paidsearch.+utm_campaign=sem2015Q3.*$/;
+}
+
+// Variation A
+function variationA() {
+    'use strict';
+    return $(".os_win > .download-link").attr({"href":"https://download.mozilla.org/?product=firefox-41.0.1-SSL-f53&os=win&lang=en-US"});
```

Comment:
Yep, the link will work once [bug 1205743](https://bugzilla.mozilla.org/show_bug.cgi?id=1205743) is closed.